starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -24,10 +24,19 @@ use warnings;
 use Inline Python => q{
 import numpy as np
 from chandra_aca.star_probs import guide_count
+from chandra_aca.transform import yagzag_to_pixels
 import Quaternion
 from Ska.quatutil import radec2yagzag
 import agasc
 
+
+def _yagzag_to_pixels(yag, zag):
+    yag = float(yag)
```

Comment:
I'll check on these casts.

Reply:
Seems to pass most regression weeks without the casts, so removing.  I'll regress test all of the final 13.0 candidate of course.